nilearn/glm/tests/test_first_level.py

```diff
@@ -211,8 +211,8 @@ def test_explicit_fixed_effects_without_mask(tmp_path):
     )
 
 
-def test_high_level_glm_with_data(tmp_path):
-    shapes, rk = ((7, 8, 7, 15), (7, 8, 7, 16)), 3
+def test_high_level_glm_with_data(tmp_path, shape_4d_default):
+    shapes, rk = [shape_4d_default, shape_4d_default], 3
```

Comment:
I am not sure if the earlier version `((7, 8, 7, 15), (7, 8, 7, 16))` did not have exactly the same value to in purpose, I mention just to make sure.